pkg/operator/staticpod/installerpod/cmd.go

```diff
@@ -83,7 +88,13 @@ func NewInstaller(ctx context.Context) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "installer",
 		Short: "Install static pod and related resources",
-		Run: func(cmd *cobra.Command, args []string) {
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// Activate logging as soon as possible, after that
+			// show flags with the final logging configuration.
+			if err := o.logs.ValidateAndApply(); err != nil {
+				return err
+			}
```

Comment:
I saw that in most of the places we are not calling `defer logs.FlushLogs()`; only in `NewControllerCommandConfig`. Can we expect the consumers will remember to call this? Can I add it in the missing places?

Reply:
although now I see that flushes were removed as well https://github.com/kubernetes/kubernetes/pull/105076/files#diff-186fff86092e78dc06e5f5943eeed276427a565d276e976cdc36939988592c3cL46, but it seems to me from looking at the code this could be skipped in last 5s window : https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/component-base/config/v1alpha1/defaults.go#L126

Reply:
Yes, I'd add Flush everywhere.

Reply:
That flush moved to `cli.Run`: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/component-base/cli/run.go#L39

Reply:
thanks, makes sense now :)